icu4c/source/data/cldr-icu-readme.txt

```diff
@@ -188,14 +196,9 @@ ant all 2>&1 | tee /tmp/cldr-newData-buildLog.txt
 
 git status
 
-# 6. Fix any errors, investigate any warnings. Some warnings are expected,
-# including  warnings for missing versions in locale names which specify some
-# collationvariants, e.g.
-#   [cldr-build] WARNING (ja_JP_TRADITIONAL): No version #??
-#   [cldr-build] WARNING (zh_TW_STROKE): No version #??
-# and warnings for some empty collation bundles, e.g.
-#   [cldr-build] WARNING (en):  warning: No collations found. Bundle will ...
-#   [cldr-build] WARNING (to):  warning: No collations found. Bundle will ...
+# 6. Fix any errors, investigate any warnings. Currently for example there are
+# a few warnings of the following form in rbnf files:
+#   [cldr-build] Warning: no version match with: $Revision☹$
```

Comment:
Just curious, do the warnings actually have the "☹" in `$Revision☹$`?

Reply:
I don't think so.For example there is a warning in the rbnf files between ar.xml and az.xml (not sure which it applies to) but they have:
```
ar <version number="$Revision: 14199 $"/>
az <version number="$Revision: 12722 $"/>
```
(with a normal space, not NBSP)

Reply:
So the emoji face is just a placeholder then? :)

Reply:
Maybe it is attempting to provide some sort of status on the rev # ? Who knows...